docs/learningLab/introToLit/acc.md

```diff
@@ -1,10 +1,22 @@
 ---
 sidebar_position: 3
 ---
-# 3. Working with Access Control Conditions 
-ACCs can be harnessed to restrict access to a particular resource (such as a file, webpage, or server). You can set an on-chain condition(s) and then the network will provision signatures and decryption keys for users that meet those conditions.
+# 3. Access Control 
+![image](../../../static/img/ll_acc.png)
 
-### **Example Conditions**
+Access control control conditions can be used to restrict access to a particular resource (such as a file, webpage, or server). You can set an on-chain conditions and then the network will provision signatures and decryption keys for users that meet those conditions.
```

Comment:
could be helpful to include a more in depth intro description of ACCs and tying them into decentralized access control / encryption more generally. Emphasizing how all these pieces fit together